_build/templates/default/sass/index.scss

```diff
@@ -551,26 +559,28 @@ textarea.x-form-field {
   color: #999 !important;
 }
 
-a.x-grid-link {
-  color: $colorSplash;
-  text-decoration: underline;
-}
-
-a.x-grid-link:hover,
-a.x-grid-link:focus {
-  text-decoration: none;
+.x-grid-link {
+    @include textLink;
+    &.simulated-link {
+        cursor: pointer;
```

Comment:
Why is this line not in the`textLink`?

Reply:
It doesn't need to be defined there, only where the "link" is not an anchor tag.